port/common/omrport.c

```diff
@@ -76,6 +76,7 @@ static OMRPortLibrary MainPortLibraryTable = {
 	omrsysinfo_processor_has_feature, /* sysinfo_processor_has_feature */
 	omrsysinfo_processor_set_feature, /* sysinfo_processor_set_feature */
 	omrsysinfo_get_processor_feature_name, /* sysinfo_get_processor_feature_name */
+	omrsysinfo_print_processor_features, /* sysinfo_print_processor_features */
```

Comment:
Looking at the other APIs that return strings I don't see any that use "print" in the name. Perhaps `omrsysinfo_get_processor_feature_string` makes more sense here?